src/components/CustomerQuota/NoQuotaCard.tsx

```diff
@@ -161,7 +161,7 @@ export const NoQuotaCard: FunctionComponent<NoQuotaCard> = ({
           {itemTransactions.length > 0 && (
             <View>
               <AppText style={{ marginBottom: size(1) }}>
-                Items {policyType === "redeem" ? "redeemed" : "purchased"}:
+                Item(s) {policyType === "REDEEM" ? "redeemed" : "purchased"}:
```

Comment:
@szenius FYI i changed the condition to all caps, i think that was done during your refactoring hehe

Reply:
Oh no need to change line 104 too

Reply:
Changing in new PR